app/views/voice/otp.xml.slim

```diff
@@ -0,0 +1,7 @@
+doctype xml
+Response
+  Say = @message
+  - if @action_url
+    Gather finishOnKey="1" action=@action_url
+      Say Press 1 to repeat this message.
```

Comment:
I need to move this to be i18ned just like the message.

Twilio supports speaking in multiple languages, but we'd have to script the choice behavior, or get a language preference from the user ahead of time and just send that: https://support.twilio.com/hc/en-us/articles/223132827-What-languages-can-the-Say-verb-speak-